crates/gpui/src/platform/linux/platform.rs

```diff
@@ -74,7 +74,7 @@ impl Default for LinuxPlatform {
 impl LinuxPlatform {
     pub(crate) fn new() -> Self {
         let wayland_display = env::var_os("WAYLAND_DISPLAY");
-        let use_wayland = wayland_display.is_some() && !wayland_display.unwrap().is_empty();
+        let use_wayland = wayland_display.is_some_and(|dp| !dp.is_empty());
```

Comment:
Can we use a more meaningful binding name here?

```suggestion
        let use_wayland = wayland_display.is_some_and(|display| !display.is_empty());
```

Reply:
I don't think it's _necessary_ (the full `wayland_display` is right here anyway), but I don't really have a preference.

Reply:
I think using `display` here would be better, just for clarity's sake.